### DISCUSSION TITLE
has appveyor disappeared?

### DISCUSSION THREAD

**Discussion by vomikan** [moneymanagerex/moneymanagerex#5404] [General]
Last build https://ci.appveyor.com/project/moneymanagerex/moneymanagerex/builds/45588186 2 day ago

![image](https://user-images.githubusercontent.com/6836805/206438124-92f3103c-6d92-4572-b54b-6b0375fe5905.png)

https://ci.appveyor.com/project/moneymanagerex/moneymanagerex/history

**Comment by whalley** [comment]
@vomikan Do you know to what account the AppVeyor build is associated?

**Reply by vomikan** [reply]
I think moneymanagerex

**Reply by n-stein** [reply]
Who manages the moneymanagerex GitHub account? They probably need to login to resolve.

Maybe the OAuth token expired

**Comment by whalley** [comment]
Not sure what happened, but I have re-associated and it now seems to be working... Though no idea how to associate with moneymanagerex user, it's tied to my AppVeyor/Github account.

https://ci.appveyor.com/project/whalley/moneymanagerex

**Reply by vomikan** [reply]
Moneymanagerex@gmail.com under control by user with phone number ********77

**Reply by n-stein** [reply]
@guanlisheng Are you the admin for moneymanagerex on AppVeyor? For some reason the CI builds stopped, trying to figure out who can login to AppVeyor to restore connection.

**Reply by whalley** [reply]
I tried the [Moneymanagerex@gmail.com](mailto:Moneymanagerex@gmail.com) address on AppVeyor and there is no account (the ********77 number is mine). 

As noted the builds should now be running as I re-associated using my GitHub account https://ci.appveyor.com/project/whalley/moneymanagerex.   

![CleanShot 2022-12-13 at 23 50 50](https://user-images.githubusercontent.com/487345/207470220-e2473ec9-7075-464b-a047-6f9c9608e69c.png)
